com.bambulab.BambuStudio.yml

```diff
@@ -98,19 +98,19 @@ modules:
       - /lib/*.la
       - /lib/pkgconfig
 
-  - name: boost
-    buildsystem: simple
-    build-commands:
-      - ./bootstrap.sh --prefix=/app --with-libraries=system,iostreams,filesystem,thread,log,locale,regex,date_time
-      - ./b2 headers
-      - ./b2 -j$FLATPAK_BUILDER_N_JOBS install variant=release cxxstd=17 --layout=system
-    sources:
-      - type: archive
-        url: https://freefr.dl.sourceforge.net/project/boost/boost/1.78.0/boost_1_78_0.zip
-        sha256: f22143b5528e081123c3c5ed437e92f648fe69748e95fa6e2bd41484e2986cc3
-    cleanup:
-      - /include
-      - /lib/cmake
+  # - name: boost
```

Comment:
Why would we want to build boost inside the BambuStudio module rather than separately, it just makes the BambuStudio build longer.